apps/src/templates/HideToolbarHelper.jsx

```diff
@@ -35,11 +26,10 @@ export default class HideToolbarHelper extends React.Component {
     // the cookie so that it isn't shown again...
     this.wasHelperShowing = false;
 
-    // ...but only do these things once.
-    this.didTrackToolbarHide = false;
-
+    this.forceShowHelper = false;
     if (window.location.search.indexOf('force_show_toolbar_helper') !== -1) {
       cookies.remove(HideToolbarHelperCookieName, {path: '/'});
+      this.forceShowHelper = true;
```

Comment:
So `force_show_toolbar_helper` now does two things: forces it to show even on non-IOS 13, and also removes the cookie?